gnc/sub8_perception/include/sub8_pcl/cv_tools.hpp

```diff
@@ -13,6 +17,8 @@
 #include <sensor_msgs/image_encodings.h>
 #include "ros/ros.h"
 
+#define VISUALIZE
```

Comment:
This shadows the #define VISUALIZE in buoy.cc and will propagate through includes. Let's remove the one in buoy.cc